### PR TITLE
[5.6] Delete duplicated config('app.debug') from Exception Handler

### PR DESCRIPTION
**Pull request by TBlindaruk**
Illuminate/Foundation/Exceptions/Handler.php refactoring about `config('app.debug')`

### CODE REVIEW

**Comment by m1guelpf** [comment]
@TBlindaruk What is exactly the gain here?

**Comment by deleugpn** [comment]
I'd say the gain is about making it easier for the possibility to change how the debugger information gets resolved.

**Comment by TBlindaruk** [comment]
@deleugpn ,
Yes.

As for me, in the future will be better to create a configuration class for getting all default config settings like `app.debug`, `app.env` etc

File: src/Illuminate/Foundation/Exceptions/Handler.php [hunk 1]
```diff
@@ -478,4 +478,12 @@ protected function isHttpException(Exception $e)
     {
         return $e instanceof HttpException;
     }
+
+    /**
+     * @return \Illuminate\Config\Repository|mixed
+     */
+    private function isDebugMode()
```
Comment (zholus) [inline src/Illuminate/Foundation/Exceptions/Handler.php]: this method should be declared like protected

Comment (TBlindaruk) [reply src/Illuminate/Foundation/Exceptions/Handler.php]: I`m not sure, that we should do change it now.

File: src/Illuminate/Foundation/Exceptions/Handler.php [hunk 2]
```diff
@@ -478,4 +478,12 @@ protected function isHttpException(Exception $e)
     {
         return $e instanceof HttpException;
     }
+
+    /**
+     * @return \Illuminate\Config\Repository|mixed
+     */
+    protected function isDebugMode()
+    {
```
Comment (tillkruss) [inline src/Illuminate/Foundation/Exceptions/Handler.php]: `inDebugMode` or `debugEnabled` would make more sense to me.

Comment (TBlindaruk) [reply src/Illuminate/Foundation/Exceptions/Handler.php]: ok, i will change to inDebugMode

**Pull request closed without merging**